File: src/main/java/ninja/hudy/infosmog/InfoSmogApplication.java
package ninja.hudy.infosmog;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
@EnableCaching
public class InfoSmogApplication {

    public static void main(String[] args) {
        SpringApplication.run(InfoSmogApplication.class, args);
    }
}

File: src/main/kotlin/pl/org/smok/infosmog/InfoSmogApplication.kt
package pl.org.smok.infosmog

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
@EnableCaching
class InfoSmogApplication

fun main(args: Array<String>) {
    SpringApplication.run(InfoSmogApplication::class.java, *args)
}
